waspc/src/Wasp/Util.hs

```diff
@@ -49,3 +51,13 @@ jsonSet _ _ _ = error "Input JSON must be an object"
 
 indent :: Int -> String -> String
 indent numSpaces = intercalate "\n" . map (replicate numSpaces ' ' ++) . splitOn "\n"
+
+concatShortPrefixAndText :: String -> String -> String
```

Comment:
maybe add an example and case of usage (when to use it)

Reply:
Sounds good, added detailed comments!

The reason why I didn't add them initially was that I thought it will be a lot of comments for a very short implementation, and tests anyway show how it works, but on the other hand I do like the idea of adding them now, tests are far away and person might not be sure where they are, while reading implementation is never fun.

I added this:
```hs
-- | Given a prefix and text, concatenates them in the following manner:
-- <prefix> <text_line_1>
--          <text_line_2>
--              ...
--          <text_line_N>
--
-- __Examples__
--
-- @
-- >>> putStrLn $ concatShortPrefixAndText "Log: " "Written to file foo.txt"
-- Log: Written to file foo.txt
-- @
--
-- @
-- >>> putStrLn $ concatShortPrefixAndText "Log: " "Written to file foo.txt\nWritten to file bar.txt"
-- Log: Written to file foo.txt
--      Written to file bar.txt
-- @
```